icons/BackButton.js

```diff
@@ -3,14 +3,15 @@ import Svg, { Path } from "react-native-svg";
 
 export default (props) => (
   <Svg
-    width={props.width}
-    height={props.height}
+    width={150}
+    height={70}
     fill="none"
     xmlns="http://www.w3.org/2000/svg"
+    {...props}
   >
     <Path
-      fillRule="evenodd"
-      clipRule="evenodd"
+      fill-rule="evenodd"
```

Comment:
this will break the app, it must be in camel case `fillRule="evenodd"` instead of fill-rule="evenodd"